src/rest_utils.cpp

```diff
@@ -24,6 +24,8 @@
 #include "tensorflow_serving/util/json_tensor.h"
 #pragma GCC diagnostic pop
 
+#include "tfs_frontend/tfstensorinfo.hpp"
```

Comment:
why have the file named "tfstensorinfo" if it is under tfs_frontend directory?
what is more, tensorinfo can be misleading due to already existing tensorinfo in main src directory, i would suggest the name to be precision utils?

Reply:
or precision conversions